test/http2_test/http2_test_server.py

```diff
@@ -0,0 +1,57 @@
+"""
+  HTTP2 Test Server. Highly experimental work in progress.
+"""
+import argparse
+import logging
+
+from twisted.internet.protocol import Factory
+from twisted.internet import endpoints, reactor
+import http2_base_server
+import test_rst_after_header
+import test_rst_after_data
+import test_rst_during_data
+import test_goaway
+import test_ping
+import test_max_streams
+
+test_case_mappings = {
+  'rst_after_header': test_rst_after_header.TestcaseRstStreamAfterHeader,
+  'rst_after_data': test_rst_after_data.TestcaseRstStreamAfterData,
+  'rst_during_data': test_rst_during_data.TestcaseRstStreamDuringData,
+  'goaway': test_goaway.TestcaseGoaway,
+  'ping': test_ping.TestcasePing,
+  'max_streams': test_max_streams.TestcaseSettingsMaxStreams,
+}
+
+class H2Factory(Factory):
+  def __init__(self, testcase):
+    logging.info('In H2Factory')
```

Comment:
Changed log message. This is useful for debugging when a new connection is made.